XenAdmin/Diagnostics/Checks/PoolCheck.cs

```diff
@@ -30,12 +30,11 @@
  */
 
 using XenAdmin.Core;
-using XenAdmin.Diagnostics.Problems;
 using XenAPI;
 
 namespace XenAdmin.Diagnostics.Checks
 {
-    public abstract class PoolCheck : Check
+    public abstract class PoolCheck : HostCheck

```

Comment:
PoolCheck should derive from Check, I can't see a need to derive from HostCheck (some of the constructors of its derivatives will need to change; currently they pass in the master, but only because they derive from HostCheck, not because the maste is needed).